src/model/wallet.ts

```diff
@@ -200,6 +205,21 @@ const authenticationPrompt = lang.t('wallet.authenticate.please');
 
 export const createdWithBiometricError = 'createdWithBiometricError';
 
+export const getHdPath = ({
+  type,
+  index,
+}: {
+  type: WalletLibraryType;
+  index: number;
+}): string => {
+  switch (type) {
+    case WalletLibraryType.ledger:
+      return `m/44'/60'/${index}'/0/0`;
```

Comment:
If you have a link to docs for this or something, mind throwing a `@see` comment at the top of this method?